### DISCUSSION TITLE
Unable to build on windows11

### DISCUSSION THREAD

**Discussion by ITSOFTNEPAL** [rustdesk/rustdesk#9168] [Q&A]
error: failed to run custom build command for `magnum-opus v0.4.0 (https://github.com/rustdesk-org/magnum-opus#5cd2bf98)`

Caused by:
  process didn't exit successfully: `E:\rustdesk\target\debug\build\magnum-opus-b75c15150879f15f\build-script-build` (exit code: 101)
  --- stdout
  cargo:info=x64-windows-static
  cargo:rustc-link-lib=static=opus
  cargo:rustc-link-search=E:\vcpkg\installed\x64-windows-static\lib
  cargo:include=E:\vcpkg\installed\x64-windows-static\include
  rerun-if-changed=C:\Users\Admin\.cargo\git\checkouts\magnum-opus-6f07f61b97016ae9\5cd2bf9\opus_ffi.h
  rerun-if-changed=E:\vcpkg\installed\x64-windows-static\include

  --- stderr
  thread 'main' panicked at C:\Users\Admin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bindgen-0.59.2\src/lib.rs:2144:31:
  Unable to find libclang: "couldn't find any valid shared libraries matching: ['clang.dll', 'libclang.dll'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to run custom build command for `scrap v0.5.0 (E:\rustdesk\libs\scrap)`

Caused by:
  process didn't exit successfully: `E:\rustdesk\target\debug\build\scrap-2edc9aa6445ce402\build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=NO_PKG_CONFIG_libyuv
  cargo:info=x64-windows-static
  cargo:rustc-link-lib=static=yuv
  cargo:rustc-link-search=E:\vcpkg\installed\x64-windows-static\lib
  cargo:include=E:\vcpkg\installed\x64-windows-static\include
  cargo:rerun-if-env-changed=NO_PKG_CONFIG_libvpx
  cargo:info=x64-windows-static
  cargo:rustc-link-lib=static=vpx
  cargo:rustc-link-search=E:\vcpkg\installed\x64-windows-static\lib
  cargo:include=E:\vcpkg\installed\x64-windows-static\include
  rerun-if-changed=E:\rustdesk\libs\scrap\src\bindings\vpx_ffi.h
  rerun-if-changed=E:\vcpkg\installed\x64-windows-static\include

  --- stderr
  thread 'main' panicked at C:\Users\Admin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bindgen-0.65.1\lib.rs:603:31:
  Unable to find libclang: "couldn't find any valid shared libraries matching: ['clang.dll', 'libclang.dll'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by rustdesk** [comment]
#763
